### DISCUSSION TITLE
Spawners emit a redstone signal

### DISCUSSION THREAD

**Discussion by refractored** [PurpurMC/Purpur#506] [Feature Requests]
Spawners emit a redstone signal when mobs are spawned from the spawner.

**Comment by BillyGalbreath** [comment]
Is this a question or a statement? lol

**Reply by refractored** [reply]
Oh sorry lol, I meant to put it as:
Can we have the ability in the config for spawners to emit a redstone signal when a mob is spawned from the spawner? I think this would be nice for automated farms. 

its 2 am for me and im quite tired rn sorry xd
